portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -47,4 +48,27 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     String serializedJSON = gson.toJson(commentsList);
     response.getWriter().println(serializedJSON);
   }
-}
\ No newline at end of file
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String username = request.getParameter("comment-username");
+    String commentText = request.getParameter("comment-input");
+    commentsList.add(Comment.create(username, commentText));
+    response.sendRedirect("/");
+  }
+
+  /**
+   * Value class for a comment, complete with a username and body text. Generated
+   * with AutoValue.
+   */
+  @AutoValue
+  abstract static class Comment {
+    static Comment create(String name, String text) {
+      return new AutoValue_DataServlet_Comment(name, text);
+    }
+
+    abstract String name();
+
+    abstract String text();
+  }
```

Comment:
Does the default toString work as you need? You can overwrite it if you want to keep the format same as before

Reply:
I didn't use that `toString()` method anyway since I was handling the format on the client-side so I can better use CSS instead of limiting myself to some string, so it's okay.